client/src/component/TicketsSidebar/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Link, Route } from 'react-router-dom';
+import './style.css';
+
+class TicketsSidebar extends Component {
+  render() {
+    return (
+      <div className="ticket-sidebar">
+        <button className="ticket-sidebar__new-ticket">
+          <i className="fas fa-plus" />
+          {'          '}
+          Ticket
+        </button>
+        <div className="ticket-sidebar__links-container">
+          <Router>
+            <div className="ticket-sidebar__links ticket-sidebar__selected">
```

Comment:
```ticket-sidebar__selected``` is used only here. It does not exist in the CSS file. It adds nothing to the element.